tests/validator/validator.cpp

```diff
@@ -897,7 +924,7 @@ static const std::string emptyMath = "<math xmlns=\"http://www.w3.org/1998/Math/
 TEST(Validator, resets)
 {
     const std::vector<std::string> expectedErrors = {
-        "Component 'comp' contains multiple resets with order '300'.",
+        // "Component 'comp' contains multiple resets with order '300'.",
```

Comment:
eg here: should be deleted as test is just wrong ... resets must have unique order within variable set, not component.